docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -41,43 +32,20 @@ Within every game (XML) file, inside the "games" folder inside the main folder o
 - `="Battleships repair at beginning of round"` >> `="Units Repair Hits Start Turn"`
 - `="Naval Bombard Casualties Return Fire Restricted"` >> `="Naval Bombard Casualties Return Fire"`
 - `="SBR Affects Unit Production"` >> `="Damage From Bombing Done To Units Instead Of Territories"`
-
-### Changes of game (XML) file special code for the above items:
-
-_(these are cases of advanced coding, so you probably don't have these within the game file)_
-
-Within every game file (XML), inside the "games" folder inside the main folder of the map, change:
-
 - `="conditionType" count="XOR"` >> `="conditionType" count="1"`
 - `="conditionType" count="-reset-XOR"` >> `="conditionType" count="-reset-1"`
-
-### Changes of already deprecated game (XML) file items:
-
-_(these items were already deprecated, so you shouldn't have had these within the game file already)_
-
-Within every game (XML) file, inside the "games" folder inside the main folder of the map, change:
-
-- `="isTwoHit" value="true"` >> `="hitPoints" value="2"`
-- `="isTwoHit" value="false"` >> `="hitPoints" value="1"`
-- `="occupiedTerrOf"` >> `="originalOwner"`
-- `="victoryCity" value="true"` >> `="victoryCity" value="1"`
-- `="victoryCity" value="false"` >> `="victoryCity" value="0"`
```

Comment:
The new document is completely missing this item. I believe it has been removed by mistake, thus needs to be added back.

Reply:
Side-note about review, it's not cool to send too many review comments. Try to group them up when possible and when there is a lot of commentary drop the least important comments.

Reply:
I guess I could have said that the same problem applies to the next line, in my previous comment, but I don't think that is the correct way to do it. They are two different items and lines, and they may have not been one next to the other, either.

Reply:
One needs to keep in mind that review is a social activity. The goal is not perfection, it is not just about the code/documentation & the text on page, you are interacting with another human. It's akin to criticizing every single article of clothing that is someone is wearing (even if each has a problem, the level of criticism conveys it's own message even if well intentioned). I'm calling this out as I fell into this same trap when first doing reviews and ignored the social aspect and had the mentality "we're going to make this the best damn thing and I'm going to call out every thing that I can see that can be improved.".